File: tests/fixtures/java/hello.java
public class Calculator {
    public int add(int a, int b) {
        return a + b;
    }

    public void print(String message) {
        System.out.println(message);
    }
}

public class Point {
    public double x;
    public double y;
}

public interface IProcessor {
    void process(String data);
}

public enum Color {
    RED,
    GREEN,
    BLUE
}

File: tests/fixtures/kotlin/hello.kt
package com.example

import java.util.List
import java.util.ArrayList

class Calculator {
    fun add(a: Int, b: Int): Int {
        return a + b
    }

    fun print(message: String) {
        println(message)
    }
}

interface IProcessor {
    fun process(data: String)
}

enum class Color {
    RED,
    GREEN,
    BLUE
}

data class Point(val x: Double, val y: Double)

object Constants {
    const val MAX_SIZE = 100
}
